src/styles/blocks/card/blockscard/cardBuy.scss

```diff
@@ -0,0 +1,24 @@
+.card__buy {
+  box-sizing: border-box;
+  height: 40px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  background: $link-bg-color;
+  color: $link-text-color;
+  border-radius: 5px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  border: 1px solid $link-bg-color;
+
+  &:hover {
+    box-sizing: border-box;
+    background-color: $link-text-color;
+    color: $link-bg-color;
+    border-color: $link-bg-color;
+    line-height: 39px;
```

Comment:
The `line-height` property should not change on hover. This can lead to unpredictable and inconsistent rendering across different browsers. If you want to achieve a specific visual effect, consider using padding, margin or transform properties instead.